internal/fusekernel/fuse_kernel.go

```diff
@@ -714,12 +716,15 @@ type InitIn struct {
 const InitInSize = int(unsafe.Sizeof(InitIn{}))
 
 type InitOut struct {
-	Major        uint32
-	Minor        uint32
-	MaxReadahead uint32
-	Flags        uint32
-	Unused       uint32
-	MaxWrite     uint32
+	Major               uint32
+	Minor               uint32
+	MaxReadahead        uint32
+	Flags               uint32
+	MaxBackground       uint16
+	CongestionThreshold uint16
+	MaxWrite            uint32
+	TimeGran            uint32
+	MaxPages            uint32
```

Comment:
This should be `MaxPages uint16` as per https://github.com/torvalds/linux/blame/master/include/uapi/linux/fuse.h, no?

I think we need (or should add nevertheless) the `unused` field as well, no?

Reply:
Fixed, thank you.